servers/src/test/java/tachyon/worker/block/allocator/BaseAllocatorTest.java

```diff
@@ -63,8 +64,10 @@ public final void before() throws Exception {
   }
 
   protected void resetManagerView() throws Exception {
+    TachyonConf tachyonConf = WorkerContext.getConf();
+    tachyonConf.merge(createTestTachyonConf());
```

Comment:
Can you leave a TODO here that we probably want to set `WorkerContext.tachyonConf`.